assets/styles/tools/_maths.scss

```diff
@@ -2,13 +2,18 @@
 // Tools / Maths
 // ==========================================================================
 
-// Removes the unit from the given number.
+// Remove the unit of a length
 //
-// @param  {number} $number The number to strip.
-// @return {number}
+// @param {Number} $number Number to remove unit from
+// @return {function<number>}
+@function strip-unit($number) {
```

Comment:
The previous `strip-units()` function is used by the `rad()` function which will need to be updated accordingly.

The function block comment should be:

```scss
// Removes any unit from the given length.
//
// @param  {number} $number The number to remove unit from.
// @return {number}

@function strip-unit($number) {
```